go/client/chat_api_test.go

```diff
@@ -697,6 +712,36 @@ var optTests = []optTest{
 	{
 		input: `{"method": "unpin", "params":{"version": 1, "options": {"channel": {"name": "alice,bob"}}}}`,
 	},
+	{
+		input: `{"id": 29, "method": "reply", "params":{"version": 1}}`,
+		err:   ErrInvalidOptions{},
+	},
```

Comment:
I'm not 100% sure if these tests are valid or applicable here. I copied them from the tests for the `edit` method.